app/components/pages/SubmissionWizard/steps/Files/index.js

```diff
@@ -29,28 +35,33 @@ const FileUploadsPage = ({
     {(uploadFile, { loading, error: uploadError }) => {
       const fieldName = 'files'
       return (
-        <FileUploads
-          conversion={{
-            converting: loading,
-            // TODO import this constant from somewhere (data model package?)
-            completed: values[fieldName].some(
-              file => file.type === 'MANUSCRIPT_SOURCE',
-            ),
-            error: uploadError,
-          }}
-          formError={errors[fieldName] && touched[fieldName]}
-          onDrop={([file]) =>
-            uploadFile({
-              variables: { file, id: values.id },
-            }).then(({ data }) => {
-              setFieldValue('meta.title', data.uploadManuscript.meta.title)
-              setFieldValue(fieldName, data.uploadManuscript.files)
-            })
-          }
-          previewUrl={`/manuscript/${values.id}`}
-          setFieldValue={setFieldValue}
-          {...props}
-        />
+        <Subscription subscription={ON_UPLOAD_PROGRESS}>
+          {({ data: uploadData, loading: uploadLoading }) => (
```

Comment:
What happens if this request fails? Does the progress just stay at zero? If so, I think that's fine. Just would like to make sure it doesn't fail in any more major way.

Reply:
I just played with this and if the initial subscription fails then the page does not load at all. The error doesn't seem to be captured in the `error` render prop as described here https://www.apollographql.com/docs/react/advanced/subscriptions.html#render-prop

Reply:
https://github.com/elifesciences/elife-xpub/issues/492